yuniql-cli/CommandLineService.cs

```diff
@@ -281,110 +281,85 @@ public int RunDestroyOption(DestroyOption opts)
+            }
         }
 
         public int RunPlatformsOption(PlatformsOption opts)
         {
             try
             {
-                string platforms = @"Supported database platforms and available samples. For specific versions, please refer to latest documentation pages.
-
-    //TODO: show released, preview, alpha, beta
-    SqlServer | Released: 
-        Supported versions: https://yuniql.io/docs/supported-platforms/
-        Usage: yuniql run -a -c <your-connection-string> --platform sqlserver
-        Samples: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-sqlserver-sample
-
-    PostgreSql | Released: 
-        Supported versions: https://yuniql.io/docs/supported-platforms/
-        Usage: yuniql run -a -c <your-connection-string> --platform postgresql
-        Samples: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-postgresql-sample
-
-    MySql | Released: 
-        Supported versions: https://yuniql.io/docs/supported-platforms/
-        Usage: yuniql run -a -c <your-connection-string> --platform mysql
-        Samples: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-mysql-sample
-
-    MariaDb | Released: 
-        Supported versions: https://yuniql.io/docs/supported-platforms/
-        Supported versions: 
-        Usage: yuniql run -a -c <your-connection-string> --platform mariadb
-        Samples: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-mysql-sample
-
-    Snowflake | Preview: 
-        Supported versions: https://yuniql.io/docs/supported-platforms/
-        Supported versions: 
-        Usage: yuniql run -a -c <your-connection-string> --platform snowflake
-        Samples: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-snowflake-sample
-
-    Redshift| Preview: 
-        Supported versions: https://yuniql.io/docs/supported-platforms/
-        Supported versions: 
-        Usage: yuniql run -a -c <your-connection-string> --platform redshift
-        Samples: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-redshift-sample
-";
-
-                Console.WriteLine(platforms);
+                Console.WriteLine(@"Supported database platforms and available samples. For specific versions, please refer to latest documentation pages.");
+                 
+                //iterate through supported db and get manifest data
+                var fields = typeof(SUPPORTED_DATABASES).GetFields();
+                foreach (var field in fields)
+                {
+                    if(field.Name.ToLower() != SUPPORTED_DATABASES.MARIADB) //extra check since mariadb is not yet fully supported[SHOULD CHANGE IN THE FUTURE]
+                    {
+                        var _dataService = _dataServiceFactory.Create(field.Name);
+                        var _manifestData =_dataService.GetManifestData();
+                        _manifestData.printData();

```

Comment:
Lets move the logic of this into the `CommandLineService` so this class remains just plain object without any behavior. How we format into the output stream would be responsibility of the CLI classes.